.github/workflows/dist.yml

```diff
@@ -36,8 +36,8 @@ jobs:
 
       - name: Install dependencies
         run: |
-          # Print only what follows ^BuildRequires in the range of lines between %if 0%{?xxxx} and %endif where xxxx is the rpm_tag
-          sed -n '/^%if 0%{?${{ matrix.rpm_tag }}}$/,/^%endif$/{s/^BuildRequires: //p}' input-leap/dist/rpm/input-leap.spec.in | xargs ${{ matrix.installer }}
+          # Replace foo >= x with "foo >= x", print only what follows ^BuildRequires in the range of lines between %if 0%{?xxxx} and %endif where xxxx is the rpm_tag
+          sed -n 's/\(\S\+ [<>=]\+ [0-9]\+.\?[0-9]\?\)/"\1"/;/^%if 0%{?${{ matrix.rpm_tag }}}$/,/^%endif$/{s/^BuildRequires: //p}' input-leap/dist/rpm/input-leap.spec.in | xargs ${{ matrix.installer }}
```

Comment:
I mentioned this in #1962 but `dnf builddep foo.spec` would look much nicer here :)

Reply:
This would require `dnf install -y 'dnf-command(builddep)'` before. 
with `input-leap/dist/rpm/input-leap.spec.in` it doesnt work though, you can't have the @ character in it.
Are you sure there is a zypper equivalent? I don't find anything.
Alternative could be to define specfile: input-leap/dist/rpm/input-leap.spec.in
and then use 
installer: sed -n '/^%if 0%{?${{ matrix.rpm_tag }}}$/,/^%endif$/{s/^BuildRequires: //p}' ${{ matrix.specfile }} | xargs zypper install -y
or 
installer 
dnf builddep foo.spec if we then have a foo.spec

Reply:
In the end, this maybe makes it more complicated even though it looks nicer

Reply:
I mentioned this on irc but `rpmspec --parse` should work to get rid of the complicated sed and it should fill in macros correctly depending on the host container. That can be a follow-up though.

Reply:
This also doesn't work with the spec.in file because of the '@' in the file when I tried

Reply:
Definitely a follow-up then...